src/queryResult/queryResultWebViewController.ts

```diff
@@ -116,6 +116,8 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
 
     private registerRpcHandlers() {
         this.registerRequestHandler("openInNewTab", async (message) => {
+            void this.createPanelController(message.uri);
```

Comment:
should be still await herre?

Reply:
nope; intentionally not awaiting it so that it the prompt doesn't wait for the tab to fully open